parts/k8s/containeraddons/1.16/kubernetesmasteraddons-omsagent-daemonset.yaml

```diff
@@ -33,7 +33,7 @@ rules:
   resources: ["pods", "events", "nodes", "namespaces", "services"]
   verbs: ["list", "get", "watch"]
 - apiGroups: ["extensions"]
-  resources: ["deployments"]
+  resources: ["replicasets"]
```

Comment:
Why change to replicasets over deployments?

Reply:
This is for our health service to build model for the workloads. This work has been done by other team member. Based on the discussion with him,  we can get metadata.ownerReferences  which used to build hierarchy  from bottom up and also handle the deployment version updates from v1 to v2 etc.